exchanges/kucoin/kucoin_test.go

```diff
@@ -1961,57 +1970,11 @@ func TestGetAuthenticatedServersInstances(t *testing.T) {
-	"Futures TradeOrdersPushDataJSON":                            `{"type": "message","topic": "/contractMarket/tradeOrders","subject": "orderChange","channelType": "private","data": {"orderId": "5cdfc138b21023a909e5ad55","symbol": "ETHUSDCM","type": "match","status": "open","matchSize": "","matchPrice": "","orderType": "limit","side": "buy","price": "3600","size": "20000","remainSize": "20001","filledSize":"20000","canceledSize": "0","tradeId": "5ce24c16b210233c36eexxxx","clientOid": "5ce24c16b210233c36ee321d","orderTime": 1545914149935808589,"oldSize ": "15000","liquidity": "maker","ts": 1545914149935808589}}`,
-	"TransactionStaticsPushDataJSON":                             `{ "topic": "/contractMarket/snapshot:ETHUSDCM","subject": "snapshot.24h","data": {"volume": 30449670,      "turnover": 845169919063,"lastPrice": 3551,       "priceChgPct": 0.0043,   "ts": 1547697294838004923}  }`,
-	"Futures EndFundingFeeSettlementPushDataJSON":                `{ "type":"message","topic": "/contract/announcement","subject": "funding.end","data": {"symbol": "ETHUSDCM",         "fundingTime": 1551770400000,"fundingRate": -0.002966,    "timestamp": 1551770410000          }}`,
-	"Futures StartFundingFeeSettlementPushDataJSON":              `{ "topic": "/contract/announcement","subject": "funding.begin","data": {"symbol": "ETHUSDCM","fundingTime": 1551770400000,"fundingRate": -0.002966,"timestamp": 1551770400000}}`,
-	"Futures FundingRatePushDataJSON":                            `{ "topic": "/contract/instrument:ETHUSDCM","subject": "funding.rate","data": {"granularity": 60000,"fundingRate": -0.002966,"timestamp": 1551770400000}}`,
-	"Futures MarkIndexPricePushDataJSON":                         `{ "topic": "/contract/instrument:ETHUSDCM","subject": "mark.index.price","data": {"granularity": 1000,"indexPrice": 4000.23,"markPrice": 4010.52,"timestamp": 1551770400000}}`,
-	"Orderbook Market Level2":                                    `{ "type": "message", "topic": "/market/level2:BTC-USDT", "subject": "trade.l2update", "data": { "changes": { "asks": [ [ "18906", "0.00331", "14103845" ], [ "18907.3", "0.58751503", "14103844" ] ], "bids": [ [ "18891.9", "0.15688", "14103847" ] ] }, "sequenceEnd": 14103847, "sequenceStart": 14103844, "symbol": "BTC-USDT", "time": 1663747970273 } }`,
-}
+var websocketPushDatas = map[string]string{}
 
 func TestPushData(t *testing.T) {
-	for key, val := range websocketPushDatas {
-		err := ku.wsHandleData([]byte(val))
-		if err != nil {
-			t.Errorf("%s: %v", key, err)
-		}
-	}
+	n := new(Kucoin)
+	sharedtestvalues.TestFixtureToDataHandler(t, ku, n, "testdata/wsHandleData.json", ku.wsHandleData)
```

Comment:
This is throwing an error: https://github.com/thrasher-corp/gocryptotrader/actions/runs/7176138279/job/19540558917#step:7:3277

Reply:
That'd be the false positive test then 😂 
Damn, sorry, that was sloppy.
At least we know it works.

Fixed.